leftwm-core/src/models/layout_manager.rs

```diff
@@ -28,7 +28,7 @@ impl LayoutManager {
             .workspaces()
             .unwrap_or_default()
             .iter()
-            .filter_map(|ws| ws.id.map(|id| (id, ws.layouts.clone())))
+            .filter_map(|ws| ws.id.map(|id| (id, ws.layouts.clone().unwrap_or_default())))
             .collect();
```

Comment:
I'm not sure if this would work better as a separate filter and map, as we don't really need to store a value if it is None. E.g.: 
```rust
.filter(|ws| ws.id.is_some() && ws.layouts.is_some())
.map(|ws| (ws.id.unwrap_or_default(), ws.layouts.clone().unwrap_or_default()));
```
or this might work:
```rust
.filter_map(|ws| (ws.id.unwrap_or_default(), ws.layouts.clone().unwrap_or_default()));
```

Reply:
Will look into this later.

Reply:
do we even need the filter then?
When trying your second suggestion I got [this clippy](https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_filter_map), so I tried:
```rust
            .map(|ws| { ( ws.id.unwrap_or_default(), ws.layouts.clone().unwrap_or_default(), ) })
```
which compiles and passes the tests. Will check in a life system in a minute..